deployment/document-understanding-solution.template

```diff
@@ -18,13 +18,14 @@ Parameters:
     Description: User email address
     Type: String
     AllowedPattern: (?:[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*|"(?:[\x01-\x08\x0b\x0c\x0e-\x1f\x21\x23-\x5b\x5d-\x7f]|\\[\x01-\x09\x0b\x0c\x0e-\x7f])*")@(?:(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?|\[(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?|[a-z0-9-]*[a-z0-9]:(?:[\x01-\x08\x0b\x0c\x0e-\x1f\x21-\x5a\x53-\x7f]|\\[\x01-\x09\x0b\x0c\x0e-\x7f])+)\])
-  KendraEnabled:
-    Description: Amazon Kendra is added as an additional capability and can be used for exploring features such as Semantic Search, Adding FAQs and Access Control Lists. This mode comes pre-loaded with a sample dataset of medical documents. 
+  SearchMode:
+    Description: DUS supports Amazon Elasticsearch for traditional search capbilities and Amazon Kendra  for features such as Semantic Search, Adding FAQs and User Context based filtering.
     Type: String
-    Default: "false"
+    Default: "AMAZON_ES_AND_KENDRA"
```

Comment:
Since we had default value for Kendra as false, shouldn't the default value for search mode be `AMAZON_ES_ONLY` ?

Reply:
Updated